apps/web-giddh/src/app/daybook/daybook.component.html

```diff
@@ -67,24 +72,41 @@ <h1 class="section_title">DayBook</h1>
             <table class="tb-head table-bordered table basic onMobileView">
                 <thead>
                     <tr>
-                        <th class="col-xs-1">Date</th>
-                        <th class="col-xs-3">Particular</th>
-                        <th class="col-xs-2" style="width: 16.5%;">Vch. Name</th>
-                        <th class="col-xs-2 text-right" style="width: 16.5%;">Vch. No.</th>
-                        <th class="col-xs-2 text-right">Debit Amt</th>
-                        <th class="col-xs-2 text-right">Credit Amt</th>
+                        <th>Date</th>
```

Comment:
Please make sure that you add this to locale folder

Reply:
Will add at the end of sprint in one go for all story